web-app/src/selection-table/SelectionTable.css

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Is this file not a duplicate from `src/selection-table/SelectionTable.css`?

Reply:
And `web-app/src/selection-table/SelectionTable.jsx` is empty.

Reply:
I am assuming this was a merge mistake, I have completely removed the web-app folder in [commit 781880c](https://github.com/SolidLabResearch/generic-data-viewer/pull/3/commits/781880c1857d597a9fd70af62c8a3756188adcb2)